tests/pytests/test_data_isis.py

```diff
@@ -0,0 +1,570 @@
+  ByteOrder           = Lsb
+  TimeDependentFrames = (10014, 1)
+  CkTableStartTime    = 217006138.29611
+  CkTableEndTime      = 217006139.96986
+  CkTableOriginalSize = 2
+  Description         = "Created by spiceinit"
+  Kernels             = ($base/kernels/spk/de405.bsp,
+                         $base/kernels/pck/pck00009.tpc)
+  SolarLongitude      = 136.56543136097
+
+  Group = Field
+    Name = J2000Q0
+    Type = Double
+    Size = 1
+  End_Group
+
+  Group = Field
+    Name = J2000Q1
+    Type = Double
+    Size = 1
+  End_Group
+
+  Group = Field
+    Name = J2000Q2
+    Type = Double
+    Size = 1
+  End_Group
+
+  Group = Field
+    Name = J2000Q3
+    Type = Double
+    Size = 1
+  End_Group
+
+  Group = Field
+    Name = AV1
+    Type = Double
+    Size = 1
+  End_Group
+
+  Group = Field
+    Name = AV2
+    Type = Double
+    Size = 1
+  End_Group
+
+  Group = Field
+    Name = AV3
+    Type = Double
+    Size = 1
+  End_Group
+
+  Group = Field
+    Name = ET
+    Type = Double
+    Size = 1
+  End_Group
+End_Object
+
+Object = Table
+  Name                 = SunPosition
+  StartByte            = 3924551
+  Bytes                = 112
+  Records              = 2
+  ByteOrder            = Lsb
+  CacheType            = Linear
+  SpkTableStartTime    = 217006138.29611
+  SpkTableEndTime      = 217006139.96986
+  SpkTableOriginalSize = 2.0
+  Description          = "Created by spiceinit"
+  Kernels              = $base/kernels/spk/de405.bsp
+
+  Group = Field
+    Name = J2000X
+    Type = Double
+    Size = 1
+  End_Group
+
+  Group = Field
+    Name = J2000Y
+    Type = Double
+    Size = 1
+  End_Group
+
+  Group = Field
+    Name = J2000Z
+    Type = Double
+    Size = 1
+  End_Group
+
+  Group = Field
+    Name = J2000XV
+    Type = Double
+    Size = 1
+  End_Group
+
+  Group = Field
+    Name = J2000YV
+    Type = Double
+    Size = 1
+  End_Group
+
+  Group = Field
+    Name = J2000ZV
+    Type = Double
+    Size = 1
+  End_Group
+
+  Group = Field
+    Name = ET
+    Type = Double
+    Size = 1
+  End_Group
+End_Object
+
+Object = History
+  Name      = IsisCube
+  StartByte = 4084583
+  Bytes     = 3483
+End_Object
+
+Object = OriginalLabel
+  Name      = IsisCube
+  StartByte = 3334447
+  Bytes     = 29992
+End_Object
+
+Object = NaifKeywords
+  BODY499_RADII                            = (3396.19, 3396.19, 3376.2)
+  INS-74699_FOCAL_LENGTH                   = 11994.9988
+  INS-74699_PIXEL_PITCH                    = 0.012
+  CLOCK_ET_-74999_848201291:62546_COMPUTED = 0000000000000040
+  INS-74699_TRANSX                         = (-82.2033, 2.0e-06, 0.012)
+  INS-74699_TRANSY                         = (11.9993, -0.012, 2.0e-06)
+  INS-74699_ITRANSS                        = (1001.28, 0.0163, -83.3333)
+  INS-74699_ITRANSL                        = (6850.08, 83.3333, 0.0163)
+  INS-74699_OD_K                           = (-0.0048509, 2.41312e-07,
+                                              -1.62369e-13)
+End_Object
+End
+"""
```

Comment:
It seems that the test label above can be significantly smaller. I also wonder if having different labels might make sense. For example, the logic above has 2 different regex statements. Do those have any edge cases that are worth testing?